apps/browser/qml/pages/components/TabItem.qml

```diff
@@ -110,31 +111,6 @@ BackgroundItem {
                     truncationMode: TruncationMode.Fade
                     color: down || activeTab ? root.highlightColor : Theme.primaryColor
                 }
-
-                IconButton {
```

Comment:
I'd keep this close button as well until housekeeping mode has also re-organizing tabs functionality.

Reply:
Ok, thanks. I understand your point now. I'll update the PR after my vacations are over, second half of August.